pkg/microservice/aslan/core/common/service/kube/service.go

```diff
@@ -235,12 +236,19 @@ func (s *Service) GetYaml(id, agentImage, aslanURL, hubURI string, useDeployment
 		return nil, err
 	}
 
+	rbacv1betaused := false
+	if kubeVersion != "" {
+		rbacv1beta, _ := version.NewVersion("1.17.0")
+		rbacv1betaused = rbacv1beta.LessThan(version.Must(version.NewVersion(kubeVersion)))
+	}
+
 	if cluster.Namespace == "" {
 		err = YamlTemplate.Execute(buffer, TemplateSchema{
 			HubAgentImage:     agentImage,
 			ClientToken:       token,
 			HubServerBaseAddr: hubBase.String(),
 			UseDeployment:     useDeployment,
+			Rbacv1beta:        rbacv1betaused,
```

Comment:
it is a common issue, we should come up with a general solution

Reply:
what about this comment?

Reply:
@27149chen  I think , since 1.17, all other source versions are stable and the probability of change is not high.

Reply:
At present, we just need to judge whether the K8S version is higher than 1.17, or do you have any suggestions

Reply:
@ysicing you are welcome to help review this pr (https://github.com/koderover/zadig/pull/688)